src/commands/skullboard.py

```diff
@@ -97,7 +97,7 @@ async def edit_or_send_skullboard_message(
         guild = self.client.get_guild(message.guild.id)
         member = guild.get_member(message.author.id)
         user_nickname = member.nick if member.nick else message.author.name
-        user_avatar_url = message.author.avatar.url
+        user_avatar_url = message.author.avatar.url if message.author.avatar else ''
```

Comment:
Replace `''` with `""` to fix the formatting action failing.

Reply:
done